samples/ComputeSharp.Sample.Shared/Helpers/Foo.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using ComputeSharp.__Internals;
+using ComputeSharp.D2D1Interop.__Internals;
+
+#pragma warning disable CS0618
+
+namespace ComputeSharp.D2D1Interop.__Internals
+{
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    [Obsolete("This interface is not intended to be used directly by user code")]
+    public interface ID2D1Shader
+    {
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete("This method is not intended to be called directly by user code")]
+        void LoadDispatchMetadata<TLoader>(ref TLoader loader, out IntPtr result)
+            where TLoader : struct, IDispatchMetadataLoader;
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete("This method is not intended to be called directly by user code")]
+        void BuildHlslString(out ArrayPoolStringBuilder builder);
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete("This method is not intended to be used directly by user code")]
+        void LoadBytecode<TLoader>(ref TLoader loader)
+            where TLoader : struct, IBytecodeLoader;
+    }
+}
+
+namespace ComputeSharp.D2D1Interop
+{
+    public interface ID2D1PixelShader : ID2D1Shader
+    {
+        float4 Execute();
+    }
+
+    
+
+    [AttributeUsage(AttributeTargets.Struct, AllowMultiple = true)]
+    public sealed class D2DInputSimpleAttribute : Attribute
+    {
+        public D2DInputSimpleAttribute(int index)
+        {
+
+        }
+
+        public int Index { get; }
+    }
+
+    [AttributeUsage(AttributeTargets.Struct, AllowMultiple = true)]
+    public sealed class D2DInputComplexAttribute : Attribute
+    {
+        public D2DInputComplexAttribute(int index)
+        {
+
+        }
+
+        public int Index { get; }
+    }
+
+    [AttributeUsage(AttributeTargets.Struct, AllowMultiple = false)]
+    public sealed class D2DRequiresScenePositionAttribute : Attribute
+    {
+    }
+
+    [AttributeUsage(AttributeTargets.Struct, AllowMultiple = false)]
+    public sealed class D2DCustomEntryAttribute : Attribute
+    {
+    }
+
+    public static class D2D1
```

Comment:
consistency ... should this be `D2D` instead of `D2D1` ? the attributes are `D2D`. it's already in a `D2D1*` named namespace 🤔

Reply:
As meentioned above, this whole file was accidentally committed.
Let's pick this up in a review thread on the correct file (`D2D1.cs`) 🙂